lib/dynamodb-integration.js

```diff
@@ -72,6 +78,27 @@ const buildPayload = (tableName, streamName, events, additionalAttributesSchema
     };
   };
 
+const isDynamoDbError = (err, data) => err || !isEmpty(data.UnprocessedItems);
+
+const sendLogsToDynamoDb = (dynamodbClient, payload, cb, times = 3) => new Promise((resolve) => {
+  dynamodbClient.send(new BatchWriteItemCommand(payload), function (err, data) {
+    debug('sent to aws, err: ', err, ' data: ', data)
+    if (isDynamoDbError(err, data) && times > 0) {
+      debug('error during batchWriteItem', err, true)
+
+      if (!isEmpty(data.UnprocessedItems)) {
+        retrySubmit(dynamodbClient, {RequestItems: data.UnprocessedItems}, cb, 3)
+      } else {
+        retrySubmit(dynamodbClient, payload, cb, 3)
```

Comment:
```suggestion
        retrySubmit(dynamodbClient, {RequestItems: data.UnprocessedItems}, cb, times-1)
      } else {
        retrySubmit(dynamodbClient, payload, cb, times-1)
```

Reply:
now you just keep trying forever

Reply:
38f44358b6c1ddbf27844bfa870841cf278509aa